src/main/java/org/jenkinsci/plugins/docker/commons/credentials/ImageNameValidator.java

```diff
@@ -99,21 +111,27 @@ public static boolean skipped() {
             return FormValidation.ok();
         }
         final String[] args = splitUserAndRepo(userAndRepo);
-        if (StringUtils.isBlank(args[0]) && StringUtils.isBlank(args[1]) && StringUtils.isBlank(args[2])) {
+        if (StringUtils.isBlank(args[0]) && StringUtils.isBlank(args[1]) && StringUtils.isBlank(args[2])
+                && StringUtils.isBlank(args[3])) {
```

Comment:
I think it should probably be
```suggestion
        if (StringUtils.isBlank(args[0]) && StringUtils.isBlank(args[1]) && (StringUtils.isBlank(args[2])
                || StringUtils.isBlank(args[3]))) {
```
?

Reply:
But the tests passes so maybe not :)

Reply:
not needed 🙃